packages/connectors/connector-kook/src/index.test.ts

```diff
@@ -128,6 +128,51 @@ describe('Kook connector', () => {
       });
     });
 
+    it('should get valid SocialUserInfo when user have not banner', async () => {
+      nock(userInfoEndpoint)
+        .get('')
+        .reply(200, {
+          code: 0,
+          message: '操作成功',
+          data: {
+            id: '364862',
+            username: 'test',
+            identify_num: '1670',
+            online: false,
+            os: 'Websocket',
+            status: 0,
+            avatar: 'https://xxx.com/assets/bot.png/icon',
```

Comment:
Do not use `xxx`, since it is usually related to pornography. Instead, we should use `abc`, `foo`, or `bar` as the dummy content.

Reply:
Replace to `img.kookapp.cn`